rosco-web/config/rosco.yml

```diff
@@ -5,7 +5,9 @@ rosco:
   configDir: /some/path/to/config/packer
   jobs:
     local:
-      timeoutMinutes: 20
+      timeoutMinutes: 30
```

Comment:
There wouldn't be any effect on the bake from leaving `orphanedJobTimeoutMinutes` as it was. That value is only compared against the amount of time since the in-flight bake was last updated. If a rosco instance is managing the bake it will be updated every 15 seconds. That is, it compares the 'staleness' of a bake against this threshold, not the lifespan of a bake.

https://github.com/spinnaker/rosco/blob/c69d724f9cdc581a9bd60c19d649272c0eb31087/rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/executor/BakePoller.groovy#L117-L122